bin/format

```diff
@@ -28,7 +28,7 @@ def parse_version(version_string):
     return int(m.group(1))
 
 def check_bin():
-    command = f"clang-format"
+    command = f"clang-format-{DEFAULT_CLANG_FORMAT_VERSION}"
```

Comment:
Hi @mlxd. `make format` doesn't work without a default version number on my machines. 
This is also compatible with the one in Lightning. What is your call?

Reply:
Thanks for the suggestion. Just update it!